Makefile

```diff
@@ -1,4 +1,24 @@
+
+IMAGE=segment/mongodb-source
+VERSION=v0.1.3-beta
+
+save-deps:
+	godep save
+
+run:
+	go run main.go mongo.go collection.go config.go description.go $(ARGS)
+
 build:
```

Comment:
should be. someone's gonna forget to add a file to go run (+ `go` doesn't respect godeps/)